etc/kernel-launchers/scala/toree-launcher/src/main/scala/launcher/ToreeLauncher.scala

```diff
@@ -153,14 +201,19 @@ object ToreeLauncher {
         // convey port number back to gateway.
         gatewaySocket = new ServerSocket(0)
         val gsJson = pidJson ++ Json.obj("comm_port" -> gatewaySocket.getLocalPort)
-        jsonContent = Json.prettyPrint(Json.toJson(gsJson))
+        // jsonContent = Json.prettyPrint(Json.toJson(gsJson))
+        jsonContent = Json.toJson(gsJson).toString()
       }
       else {
-        jsonContent = Json.prettyPrint(Json.toJson(pidJson))
+        // jsonContent = Json.prettyPrint(Json.toJson(pidJson))
```

Comment:
Could you please delete code that is commented out - unless there's a comment indicating why the code is removed?  (Same for L 161 and 204)